src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -56,15 +65,41 @@
         event.target.closest('form').submit();
     };
     const toggleDisabledStateOnApplyBtn = () => {
+        let isLastModifiedRangeSelected = true;
+        let isCreatedRangeSelected = true;
         const contentTypeOption = contentTypeSelect.querySelector('option');
         const isContentTypeSelected = contentTypeOption.innerHTML !== contentTypeOption.dataset.default;
         const isSectionSelected = sectionSelect ? !!sectionSelect.value : false;
         const isModifiedSelected = !!lastModifiedSelect.value;
         const isCreatedSelected = !!lastCreatedSelect.value;
         const isCreatorSelected = !!searchCreatorInput.value;
         const isSubtreeSelected = !!subtreeInput.value.trim().length;
+
+        if (lastModifiedSelect.value === 'custom_range') {
+            const flatPickrInstance = lastModifiedDateRange._flatpickr;
```

Comment:
Do we have to rely on the flatpickr instance? Cannot we just check if `periodSelector` and `endSelector` have values? I think we could replace the `isModifiedSelected` and `isCreatedSelected` with this check.